src/components/Collage/index.tsx

```diff
@@ -16,7 +16,7 @@ export const Collage: React.FunctionComponent<ICollage> = ({ images, title, cta,
   }
 
   return (
-    <Section {...sectionProps}>
+    <Section {...sectionProps} noPaddingT>
```

Comment:
I suggest passing this prop in from wherever you're using it. If you add `noPaddingT` here, it will apply everywhere this component is used

```suggestion
    <Section {...sectionProps}>
```